gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -133,6 +136,10 @@ mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
 <% config_file = config_file_path.split('/').last -%>
 mv %{buildroot}%{gem_instdir}/<%= config_file_path %> \
    %{buildroot}%{foreman_proxy_settingsd_dir}/<%= config_file.chomp('.example') %>
+<% if has_cronjob -%>
+mkdir -p %{buildroot}%{?scl:%_root_sysconfdir}%{!?scl:%_sysconfdir}/cron.d/
```

Comment:
On line 13 we ensure the `%_root_sysconfdir` macro is defined. Same below
```suggestion
mkdir -p %{buildroot}%{%_root_sysconfdir}/cron.d/
```

Reply:
I changed the other template too so both have the same logic.